src/containers/App.js

```diff
@@ -338,8 +338,12 @@ class App extends Component {
     }
 
     if (
-      prevProps.accountStatus === Status.LOADING &&
-      this.props.accountStatus !== Status.LOADING
+      (prevProps.accountStatus === Status.LOADING &&
```

Comment:
since this was a pain to figure out, worth a comment or self-explaining variables

const accountStatusLoaded = prevProps.accountStatus === Status.LOADING && ...
const onSignUpPage = matchPath

if (accountStatusLoaded || onSignUpPage)